organizations/middleware.py

```diff
@@ -13,31 +14,41 @@ def get_organization(request):
         """
         Returns the value of the `ORGANIZATION` HTTP header
         """
-        org = request.META.get("HTTP_ORGANIZATION", get_public_schema_name())
+        org = request.META.get("HTTP_ORGANIZATION", DEFAULT_TENANT_SHORTCODE)
         if not org:
-            return get_public_schema_name()
+            return DEFAULT_TENANT_SHORTCODE
```

Comment:
If we're returning the shortcode, shouldn't the name of this function be `get_organization_shortcode` instead of `get_organization`?